project/strapi.py

```diff
@@ -0,0 +1,29 @@
+from . import BaseProject
+
+class Strapi(BaseProject):
+
+    # Keeps package-lock.json out of the repo as recommended. See project/nextjs.py for more info.
```

Comment:
I'm not wild about comments in one template referring to another.  If you want to provide more detailed info than fits here, that should be in a separate nodes.md file, either in the project or, maybe something top level.